servers.json

```diff
@@ -1438,6 +1438,14 @@
         ]
     },
     {
+        "id": "hamstersmp",
```

Comment:
This is improperly formatted. Please refer to some JSON formatting documentation.